solana/pyth2wormhole/client/src/attestation_cfg.rs

```diff
@@ -334,44 +483,109 @@ mod tests {
     }
 
     #[test]
-    fn test_add_symbols_works() -> Result<(), ErrBox> {
-        let empty_config = AttestationConfig {
-            min_msg_reuse_interval_ms:    1000,
-            max_msg_accounts:             100,
-            min_rpc_interval_ms:          42422,
-            mapping_addr:                 None,
-            mapping_reload_interval_mins: 42,
-            symbol_groups:                vec![],
-        };
+    fn test_instantiate_batches() -> Result<(), ErrBox> {
```

Comment:
This test does not assert desired behavior edge cases:
- Group already has the same symbol
- No name group matches this symbol (default group)
- brand new symbol is added to the right place